src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp

```diff
@@ -397,6 +424,21 @@ class instance_stratholme : public InstanceMapScript
         {
             events.Update(diff);
 
+            if (GetData(TYPE_BARTHILAS_RUN) == NOT_STARTED)
+            {
+                if (GameObject* go = instance->GetGameObject(_door_service_entrance))
+                {
+                    if (go->GetGoState() == GO_STATE_ACTIVE)
+                    {
+                        if (Creature* barthilas = instance->GetCreature(_barthilasGUID))
+                        {
+                            if (barthilas->IsAlive() && !barthilas->IsInCombat())
+                                SetData(TYPE_BARTHILAS_RUN, IN_PROGRESS);
```

Comment:
```suggestion
                            if (barthilas->IsAlive() && !barthilas->IsInCombat())
                            {
                                SetData(TYPE_BARTHILAS_RUN, IN_PROGRESS);
                            }
```
add brackets here too please